src/private-key.ts

```diff
@@ -22,7 +25,22 @@ function findPrivateKey (filepath: string): Buffer | string {
     return fs.readFileSync(filepath)
   }
   if (process.env.PRIVATE_KEY) {
-    return process.env.PRIVATE_KEY.replace(/\\n/g, '\n')
+    const cert = process.env.PRIVATE_KEY
+    const begin = '-----BEGIN RSA PRIVATE KEY-----'
+    const end = '-----END RSA PRIVATE KEY-----'
```

Comment:
Is there a scenario where one of these won't be included? I assume it'll be an invalid key if one is not, but just curious for the purposes of content detection if only one of these checks is sufficient (or even simplifying it to `RSA PRIVATE KEY`).

Reply:
Just covering edge cases where a key wasn't copypasta'd correctly. This is probably my `.enterprise me` coming out, where I've seen people accidentally backspace or not select all the characters. it would be an invalid key at that point and would fail the verify stage, but then it's throwing OpenSSL errors that can be difficult to understand.